vignettes/combine-cut-interact.Rmd

```diff
@@ -0,0 +1,140 @@
+  ))
+```
+```{r, eval = FALSE}
+categories(ds$age3)
+```
+```{r, echo = FALSE}
+age3.cats
+```
+And now we have a new variable with the alias `age3`, the name "Age (3 categories)", and a category that combines 18 to 44 year olds.
+
+Note how this created an entirely new variable that exists in the dataset now, and so we can use it just like any other variable in Crunch: as a filter, etc. If we want to hide the variable we started with—"Age (4 categories)"—because we no longer need it, that is fine and will not affect our new variable.
+
+## Categorical Array Type Variables
+Similar to above with Categorical type variables, for Categorical Arrays we can combine together the answer choices that a person gave using the `combine()` function. 
+
+Let's take the variable "Issue Importance (categorical array)", which has the alias `imiss` and 11 subvariables with 4 categories: Very Important, Somewhat Important, Not very Important, and Unimportant. We would like to create a new Categorical Array variable that combines the two Important categories together and another that combines the two Not Important.
+
+```{r, eval = FALSE}
+categories(ds$imiss)
+```
+```{r, echo = FALSE}
+imiss.cats
+```
+```{r, eval = FALSE}
+ds$imiss_topboxes <- combine(ds$imiss, name="Issue Importance (Top Boxes)", 
+  combinations=list(
+    list(name="Important", categories=c("Very Important", "Somewhat Important")),
+    list(name="Not Important", categories=c("Not very Important", "Unimportant"))
+  ))
+```
+```{r, eval = FALSE}
+categories(ds$imiss_topboxes)
+```
+```{r, echo = FALSE}
+imiss_topboxes.cats
+```
+We have created a new Categorical Array type variable with the alias `imiss_topboxes`, the name "Issue Importance (Top Boxes)", and 2 categories instead of the original variable's 4.
+
+
+## Multiple Response Type Variables
+At first it might not seem that we can use the `combine()` function with Multiple Response type variables because each subvariable in the multiple response has already been reduced down to the categories that are "selected" or "not selected". However, there is an option that allows us to combine the subvariables (aka responses) in a multiple response similar to how we combined the categories in a categorical variable.  
+
+```{r, eval = FALSE}
+ds$boap
+```
+```{r, echo = FALSE}
+show_boap
+```
+```{r, eval = FALSE}
+ds$boap_combined <- combine(ds$boap, name="Approval of Obama on issues (Combined Subvariables)", 
+  combinations=list(
+    list(name="All Others", responses=c('boap_2', 'boap_3', 'boap_4', 'boap_5', 'boap_6',
+      'boap_7', 'boap_8', 'boap_9', 'boap_10', 'boap_11'))
+  ))
+```
+```{r, eval = FALSE}
+ds$boap_combined
+```
+```{r, echo = FALSE}
+show_boap_combined
+```
+
+We have created a new Multiple Response type variable with the alias `boap_combined`, the name "Approval of Obama on issues (Combined Subvariables)", which has 4 subvariables instead of the original 13.
+
+# Re-Combining Variables
```

Comment:
Again, I think this might be ok just called "Combining variables"

Reply:
yup